index.html

```diff
@@ -13,6 +13,7 @@
   <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
 
   <div class="container">
+    <div style="display: none" id="orientation_button">Vertical</div>
```

Comment:
This PR actually adds some new functionality along with refactoring - so technically this does two things at once... Acceptable, but a little convoluted!